packages/governance/src/committee.js

```diff
@@ -40,13 +43,26 @@ const { ceilDivide } = natSafeMath;
  *   committeeSize: number,
  * }>} zcf
  * @param {{ storageNode: ERef<StorageNode>, marshaller: ERef<Marshaller>}} privateArgs
+ * @param {import('@agoric/vat-data').Baggage} baggage
  * @returns {{creatorFacet: CommitteeElectorateCreatorFacet, publicFacet: CommitteeElectoratePublic}}
  */
-const start = (zcf, privateArgs) => {
+export const prepare = (zcf, privateArgs, baggage) => {
   /** @type {MapStore<Handle<'Question'>, import('./electorateTools.js').QuestionRecord>} */
-  const allQuestions = makeScalarMapStore('Question');
-  assert(privateArgs?.storageNode, 'Missing storageNode');
-  assert(privateArgs?.marshaller, 'Missing marshaller');
+  const allQuestions = provideDurableMapStore(baggage, 'Question');
+
+  // CRUCIAL: voteCap carries the ability to cast votes for any voter at
+  // any weight. It's wrapped here and given to the voter.
+  //
+  // Ensure that the voter can't get access to the unwrapped voteCap, and
+  // has no control over the voteHandle or weight
+  /** @type {Parameters<typeof prepareVoterKit>[1]['submitVote']} */
+  const submitVote = (questionHandle, voterHandle, positions) => {
+    const { voteCap } = allQuestions.get(questionHandle);
+    return E(voteCap).submitVote(voterHandle, positions);
```

Comment:
I prefer that the weight be explicit rather than defaulted, just so it's clear who's responsible for it.

```suggestion
    return E(voteCap).submitVote(voterHandle, positions, 1n);
```